03-bundling/06-vite/03-sass/README.md

```diff
@@ -56,16 +56,18 @@ Install [Node.js and npm](https://nodejs.org/en/) (18 / 20+) if they are not alr
   💥 Notice we got an error here:
 
   ```cmd
-  [vite] Internal server error: Preprocessor dependency "sass" not found. Did you install it?
+  [vite] Internal server error: Preprocessor dependency "sass-embedded" not found. Did you install it? Try `npm install -D sass-embedded`.
     Plugin: vite:css
   ```
 
-  👍🏼 In fact is a `vite` reminder to install `sass` preprocessor dependency. We forgot to do it!
+  👍🏼 In fact is a `vite` reminder to install `sass-embedded` preprocessor dependency. We forgot to do it!
 
-- Let's go then! Just add `sass` to our project as development dependency. First, stop the server and then run:
+  > Note `sass-embedded` has same API that `sass` package but it's faster.
```

Comment:
From [sass-embedded docs](https://github.com/sass/embedded-host-node?tab=readme-ov-file#embedded-sass-host):
> This package is an alternative to the [sass](https://www.npmjs.com/package/sass) package. It supports the same JS API as sass and is maintained by the same team, but where the sass package is pure JavaScript, sass-embedded is instead a JavaScript wrapper around a native Dart executable. This means sass-embedded will generally be much faster especially for large Sass compilations, but it can only be installed on the platforms that Dart supports: Windows, Mac OS, and Linux.